dgds_backend/config_data/datasets.json

```diff
@@ -16,8 +16,207 @@
     }
   ],
   "datasets": [
+    {
+      "bbox": [[3.0, 50.0], [7.0, 57.0]],
+      "scope": "regional",
```

Comment:
Does the scope belong in each dataset? or each theme?

Reply:
What a theme is isn't defined yet (theme, region, dataset). So this allows us to implement something that works, even if there are overlapping themes. A new theme has been set, which could at a later point be "scoped" as well. For now, this scope allows zooming functionality.